ticket.py

```diff
@@ -161,4 +161,95 @@ def acumulado_por_cabina(v):
         cant[pos] += 1
 
 
-# Encontrar patente cargada por usuario
\ No newline at end of file
+# Encontrar patente cargada por usuario
+
+
+
+#Punto 7
+def calcular_importe(cabina_pais, tipo_vehiculo, forma_pago): 
+    """
+    Calcula el importe final a pagar en el peaje
+    :param cabina: <str> (0: Argentina - 1: Bolivia - 2: Brasil - 3: Paraguay - 4: Uruguay)
+    :param tipo_vehiculo: <str> (0: motocicleta, 1: automóvil, 2:camión)
+    :param forma_pago: <str> (1: manual, 2 telepeaje)
+    :return: <float> importe a pagar
+    """
+    peaje_general = 300
+    peaje_bolivia = 200
+    peaje_brasil = 400
+
+    importe_final = 0
+
+    if cabina_pais == 1:
+        importe_final += peaje_bolivia
+    elif cabina_pais == 2:
+        importe_final += peaje_brasil
+    else:
+        importe_final += peaje_general
+
+    if tipo_vehiculo == 0:
+        # Moto: descuento de 50%
+        importe_final *= 0.5
+    elif tipo_vehiculo == 2:
+        # Camión: recarga de 60%
+        importe_final *= 1.6
+    # Auto: sin recargos
+
+    # Descuento del 10% por telepeaje
+    if forma_pago == 2:
+        importe_final *= 0.9
+
+    return int(importe_final)
+
+def calcular_vector_acumulador(v):
+  v_acumulador = [0] * 3
+  for ticket in v:
+    v_acumulador[ticket.tipo_vehiculo] += calcular_importe(ticket.pais_cabina, ticket.tipo_vehiculo, ticket.forma_pago)
+  return v_acumulador
+
+def vehiculo_mayor_acumuladο(v):
+  v_acumulador = v
+  monto_mayor, monto_total, porcentaje = 0, 0, 0
+  vehiculo_mayor = 4
+  i = 0
+  #Recorre el vector acumulador y determina qué tipo de vehiculo tiene le monto mayor
+  for tipo in v_acumulador:
+    monto_total += tipo
+    if tipo > monto_mayor:
+      monto_mayor = tipo
+      vehiculo_mayor = i
+    i+=1
+  if monto_mayor != 0:
+    porcentaje = monto_mayor * 100  / monto_total
+  
+  return [vehiculo_mayor, porcentaje]
+  
+def mostrar_acumulado_por_vehiculo (v):
+  if not v:
+    print(f"\nPor favor, registre previamente algun ticket.")
+  else:
+    v_acumulador = calcular_vector_acumulador(v)
+    print(f"\n*** Monto acumulado por cada tipo de vehiculo ***\n\n"
+          f"Motocicletas: ${v_acumulador[0]}\n"
+          f"Automoviles: ${v_acumulador[1]}\n"
+          f"Camiones: ${v_acumulador[2]}\n")
```

Comment:
Hacer una secuencia asociada al índice del v_acumulador para imprimir los tipos de vehiculos en un for loop